tests/ft_integration/Tribe/Tickets/__snapshots__/AttendeesTest__test_display__single event in series with Single Tickets, RSVPs and Series Passes__0.snapshot.html

```diff
@@ -84,7 +84,7 @@ <h3 class="tec-tickets__admin-attendees-overview-title">
 							#POST_ID						</span>
 					</div>
 					<div class="tec-tickets__admin-attendees-overview-ticket-type-list-item-stat">
-						3 issued (97 available)					</div>
```

Comment:
Why these changes?

Reply:
These are not related to the changes for the feature but to stabilize the tests. There were some random fails as the `menu_order` was not set for them.

Reply:
They just moved places. No real data changed